luigi/task.py

```diff
@@ -566,16 +566,17 @@ def _without_unpicklable_properties(self):
             class DummyTask(luigi):
 
                 def _dump(self):
-                    with self._without_unpicklable_properties():
+                    with self.no_unpicklable_properties():
                         pickle.dumps(self)
 
         """
         unpicklable_properties = ('set_tracking_url', 'set_status_message')
+        placeholder_during_pickling = None
```

Comment:
Nonono....

My point was that when somebody is debugging why something is not working. and they suddenly see that  `mytask.set_tracking_url` is `None`, they will be clueless. I basically suggest you edit this to 

```
placeholder_during_pickling = 'placeholder_during_pickling'
```

But of course inline it ^^